common/services/prismic/api.js

```diff
@@ -20,7 +19,8 @@ function periodicallyUpdatePrismic() {
     try {
       memoizedPrismic = await Prismic.getApi(apiUri);
     } catch (error) {
-      Raven.captureException(new Error(`Prismic error: ${error}`));
+      // FIXME: this is tipping us over our Sentry quota.
```

Comment:
```suggestion
      Raven.captureMessage('Prismic: API failed to fetch', 'info')
```

Reply:
I _think_ this is still going to count towards our quota though? I couldn't see written anywhere that it wouldn't.

I was wondering if filtering in the [`ignoreErrors` array](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/views/pages/_app.js#L345) would be slightly better – then we could at least keep a record of everything we're ignoring in one place, instead of various `FIXME`s peppered around the codebase?

Reply:
As per the conversation, let's leave it and bump up the quota.

Reply:
but also

> we shouldn't pay for a subscription just to remove a limit on alerts that we're not working to address

I'm not addressing this because I don't know what (beyond unknown networking issues) are causing the `Failed to fetch` error. Not sure how to get any more information about this?